configure

```diff
@@ -26,7 +26,8 @@ import nodedownload
 parser = optparse.OptionParser()
 
 valid_os = ('win', 'mac', 'solaris', 'freebsd', 'openbsd', 'linux', 'android')
-valid_arch = ('arm', 'arm64', 'ia32', 'mips', 'mipsel', 'x32', 'x64', 'x86')
+valid_arch = ('arm', 'arm64', 'ia32', 'mips', 'mipsel', 'x32', 'x64', 'ppc',
```

Comment:
I'm happy to sort but a question so that I understand for the future.  The other valid_x  definitions (ex valid_os above) don't look like they are sorted.  Is it that all lists like this should be sorted but we just are not there due to history or that its this specific list that should be sorted for some reason ?